### ISSUE TITLE
trying with single location

### ISSUE THREAD

**Issue by pnouvellet** [mrc-ide/EpiEstim#121]
hi 
estimate_joint wont work with 1 single location, due to
incid_imported <- incid
with incid_imported  initially NULL
 in line 548 of gibbs_draw.r
we need to force keeping the array to3 dimensions and not collapse to a 2d matrix...

**Comment by sangeetabhatia03** [comment]
yes i believe it can be closed. sorry I can't understand my own message 🤦‍♀️
